cmd/generator/main.go

```diff
@@ -31,10 +31,11 @@ import (
 )
 
 func init() {
-	generatorCmd.PersistentFlags().String("github-token", "", "The optional GitHub token for API requests.")
 	generatorCmd.PersistentFlags().Bool("debug", false, "Whether to output debug logs.")
-	generatorCmd.PersistentFlags().Bool("include-pre-release", false, "Whether to include pre-release versions.")
-	generatorCmd.PersistentFlags().String("existing", "", "An existing plugins.json to help streamline incremental updates.")
+	generatorCmd.PersistentFlags().String("existing", "plugins.json", "An existing plugins.json to help streamline incremental updates.")
```

Comment:
Building the whole database from scratch doesn't make sense in most cases. Hence, it now opt in.

Reply:
Related to eliminating `jq` and `sponge`, I wonder if we could change this to be just a `database` flag, with the expectation of reading and writing directly to that file. It could still default to `plugins.json`.

Reply:
:+1: for that suggestion. With a `database` flag in place, would there still be the possibility to output to std out? Is this needed?

Reply:
Filled https://mattermost.atlassian.net/browse/MM-21893